mysql-test/suite/rocksdb/t/index_merge_rocksdb.test

```diff
@@ -0,0 +1,93 @@
+--source include/have_rocksdb.inc
+# Skiping this test from Valgrind execution as per Bug-14627884
```

Comment:
Without knowing what "14627884" is, I'd try running this test under Valgrind and possibly removing not_valgrind.inc if it appears to work then. Depending on the runtime, it might need not_valgrind_without_big.inc instead